README.md

```diff
@@ -478,6 +478,7 @@ Hurray! You successfully made a contribution! 🎉
   - [Raveena Bhasin](https://github.com/RaveenaBhasin)
   - [Remco Halman](https://github.com/remcohalman)
   - [Riadh Laabidi](https://github.com/riadh26)
+  - [Rik Chatterjee](https://github.com/Bluetoothworks)
```

Comment:
Hey, um could you arrange your name in alphabetical order?
This should be after `- [Rick Flores](https://github.com/RICK-FLORES)`